src/Widgets/FirmwareUpdateRow.vala

```diff
@@ -20,19 +20,23 @@
 */
 
 public class About.Widgets.FirmwareUpdateRow : Gtk.ListBoxRow {
-    public Fwupd.Client client { get; construct set; }
     public Fwupd.Device device { get; construct set; }
-    public bool is_updatable { get; private set; default = false; }
+    public Fwupd.Release? release { get; construct set; }
 
-    public signal void on_update_start ();
-    public signal void on_update_end ();
+    public bool is_updatable {
+        get {
+            return release != null && device.get_version () != release.get_version ();
+        }
+    }
```

Comment:
Why was this done? We already check this on construct, this change means we have to run this comparison every time this property is used